src/main/java/com/contrastsecurity/sdk/scan/CodeArtifact.java

```diff
@@ -40,6 +41,10 @@
   /** @return filename */
   String filename();
 
+  @Nullable
+  /** @return metadata filename */
```

Comment:
can you put the annotation bellow the comment?

Reply:
@bryanateoan consider adding Checkstyle to this project to catch things like this. See https://checkstyle.sourceforge.io/config_javadoc.html#InvalidJavadocPosition